### PR TITLE
[TS-55] Follow private account

### PR DESCRIPTION
**Pull request by Ming424**
Video URL: https://youtu.be/CV6HjeJbGq8
(there is an error at reject_follow_request, it is fixed afterware)

+ Frontend for request
+ Integration of request API
+ PIE chart
+ Search user
+ E2E

### CODE REVIEW

File: src/components/NavBar/NavBar.vue
Comment (GadoliniumPotassium) [inline src/components/NavBar/NavBar.vue:226]: remove this

Comment (Ming424) [reply src/components/NavBar/NavBar.vue]: done

**Review by GadoliniumPotassium — APPROVED** [review]

**Comment by GadoliniumPotassium** [comment]
you have to deal with conflicting files before merging which will dismiss our reviews

**Review by itsShaVawn — APPROVED** [review]

**Pull request merged**
